src/lib/holocene/dropdown-button/index.svelte

```diff
@@ -5,31 +5,46 @@
 
   export let label: string;
   export let icon: IconName = undefined;
-  export let showDropdown: boolean = false;
+  export let readonly: boolean = false;
```

Comment:
When you set defaults like this, you don't need the type (i.e. it knows the type based on the default value). Just fyi so typescript warnings don't bark at ya.

Reply:
Nice ok, I'm used to explicitly typing everything. Surprised TS would yell at you for being safe 😅